internal/kafka/internal/handlers/validation.go

```diff
@@ -36,14 +35,29 @@ var ClusterIdLength = 32
 
 const minimunNumberOfNodesForTheKafkaMachinePool = 3
 
-func ValidateBillingModel(kafkaRequestPayload *public.KafkaRequestPayload) handlers.Validate {
+func validateKafkaBillingModel(ctx context.Context, kafkaService services.KafkaService, kafkaConfig *config.KafkaConfig, kafkaRequestPayload *public.KafkaRequestPayload) handlers.Validate {
 	return func() *errors.ServiceError {
-		// the billing model can only be either standard or marketplace
-		billingModel := shared.SafeString(kafkaRequestPayload.BillingModel)
-		if billingModel != "" && billingModel != string(v1.BillingModelStandard) && billingModel != string(v1.BillingModelMarketplace) {
-			return errors.InvalidBillingAccount("invalid billing model: %s, only %v and %v are allowed", billingModel,
-				v1.BillingModelStandard, v1.BillingModelMarketplace)
+		// No explicitly set kafka billing mode is allowed for now, in which case
+		// an implementation-defined default is chosen
+		if shared.StringEmpty(kafkaRequestPayload.BillingModel) {
+			return nil
+		}
```

Comment:
I believe the implementation is correct, but I don't understand the comment

Reply:
It just means that kafka billing model does not need to be provided explicitely. In case it is not provided then we choose a default (in the implementation)